### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by robinshine**
# title1

### CODE REVIEW

**Comment by robinshine** [comment]
#9

**Comment by robinshine** [comment]
Maybe having #9 is a good idea

**Comment by robinshine** [comment]
#12

**Comment by robinshine** [comment]
#8

**Comment by robinshine** [comment]
#1

**Comment by 2625265146** [comment]
image

File: file [hunk 1]
```diff
@@ -1,16 +1,18 @@
-Map<Integer, Integer> oldLinesMap = new HashMap<>();
-Map<Integer, Integer> newLinesMap = new HashMap<>();
+public contextSizes = new WebStart;
+Map<Integer, Integer> oldLinesMap2 = new HashMap<>();
```
Comment (robinshine) [inline file]: Hi this is what I mean

File: file [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+Map<Integer, Integer> newLinesMap2 = new HashMap<>();
```
Comment (robinshine) [inline file]: may i help you 
this is what you want?

File: file [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
 
-int index = 0;
 for (DiffLine diff: change.getDiffLines()) {
```
Comment (robinshine) [inline file]: something works like a charm
something works like a charm
something works like a charm
something works like a charm
something works like a charm
something works like a charm

**Comment by robinshine** [comment]
#5

**Comment by robinshine** [comment]
@2625265146  hello how

**Comment by robinshine** [comment]
hello world

**Comment by robinshine** [comment]
#1

**Comment by 2625265146** [comment]
#1

**Pull request closed without merging**